Mata/TypeExtensions.cs

```diff
@@ -5,6 +5,7 @@
 namespace Mata
 {
     using System;
+    using System.Reflection;
 
     public static class TypeExtensions
     {

```

Comment:
This is a great place to put the other extension methods I mentioned in an earler comment.